sdk/ml/azure-ai-ml/azure/ai/ml/_schema/schedule/create_job.py

```diff
@@ -95,6 +96,15 @@ def pre_load(self, data, **kwargs):  # pylint: disable=unused-argument
     def make(self, data: dict, **kwargs) -> "Job":
         from azure.ai.ml.entities import Job
 
+        def _update_data_to_job_dict(job_dict, data):
```

Comment:
NIT: make it a shared util func?

Reply:
and maybe return the base dict instead of None as updated dict is usually what we'll refer to soon